wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -173,8 +173,13 @@ public static Command defer(Supplier<Command> supplier, Set<Subsystem> requireme
    *
    * @param supplier the command supplier
    * @return the command
+   * @deprecated The ProxyCommand supplier constructor has been deprecated in favor of directly
+   *     proxying a {@link DeferredCommand}, see ProxyCommand documentaion for more details. As a
+   *     replacement, consider using `defer(supplier).asProxy()`.
    * @see ProxyCommand
    */
+  @Deprecated(since = "2024", forRemoval = true)
+  @SuppressWarnings("removal")
```

Comment:
Is this necessary? Doesn't the compiler allow using deprecated symbols inside of deprecated symbols?

Reply:
Before I suppressed it, builds would fail with the reason "removal".